packages/xod-tabtest/src/Tabtest.re

```diff
@@ -170,6 +171,7 @@ module TestCase = {
     | Boolean(true) => "true"
     | Boolean(false) => "false"
     | NaN => "NAN"
+    | Byte(x) => string_of_int(x)
```

Comment:
I guess this line is not necessary since `Byte(x)` will be correctly handled by the last clause.